app/src/AddonManager.cpp

```diff
@@ -29,6 +30,13 @@
 
 AddonManager globalAddonManager;
 
+const QString AddonManager::addonModelNameProperty = "name";
+const QString AddonManager::addonModelDescriptionProperty = "description";
+const QString AddonManager::addonModelIconFileNameProperty = "iconFileName";
+const QString AddonManager::addonModelRepositoryUrlProperty = "repositoryUrl";
+const QString AddonManager::addonModelVersionProperty = "version";
+const QString AddonManager::addonModelQtMajorProperty = "qtMajor";
```

Comment:
```suggestion
const QString AddonManager::addonModelQtMajorProperty = "qtMajor";

AddonManager globalAddonManager;
```
One detail before I can merge this.
globalAddonManager should be defined after the property strings because the application will crash when they're accessed from the AddonManager constructor.

Reply:
It didn't help, crash exists.